ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/testdata/templates/Template1TestData.kt

```diff
@@ -9,7 +9,7 @@ import java.net.URL
 
 internal val TestData.template1: PaywallData
     get() = PaywallData(
-        templateName = PaywallTemplate.TEMPLATE_1.name,
+        templateName = PaywallTemplate.TEMPLATE_1.id,
```

Comment:
Yikes, this is so easy to get wrong, considering we call this template _name_. I guess a consequence of enums having an implicit "name" property.